firmware/quadruna/BMS/src/app/app_heartbeatMonitors.c

```diff
@@ -0,0 +1,24 @@
+//
+// Created by jessi on 2024-12-28.
+//
+
+#include "app_heartbeatMonitors.h"
+#include "app_canRx.h"
+#include "app_canAlerts.h"
+#include "app_canTx.h"
+
+static HeartbeatMonitorBoard boards[1] = {
```

Comment:
This is excellent - so much better than the the previous refactor. One nit though - can we call this `heartbeat_boards`?